test/integration/guest_user_test.rb

```diff
@@ -5,11 +5,27 @@ class GuestUserTest < ActionDispatch::IntegrationTest
   include FactoryGirl::Syntax::Methods
 
   test "a guest user can view a home page" do
-    category = create(:category)
+    project = create(:project)
+
+    visit root_path
+
+    assert page.has_content?(project.categories.first.name)
+  end
 
+  test "a guest user can see all the categories for active projects" do
+    projects = []
+    4.times { |x| projects << create(:project) }
```

Comment:
Unused block argument - `x`. You can omit the argument if you don't care about it.